RecoHI/HiTracking/python/hiPixelLessStep_cff.py

```diff
@@ -0,0 +1,126 @@
+import FWCore.ParameterSet.Config as cms
+import RecoTracker.IterativeTracking.iterativeTkConfig as _cfg
+from RecoTracker.IterativeTracking.PixelLessStep_cff import *
+from HIPixelTripletSeeds_cff import *
+from HIPixel3PrimTracks_cfi import *
+
+##########################################################################
+# Large impact parameter tracking using TIB/TID/TEC stereo layer seeding #
+##########################################################################
+
+hiPixelLessStepClusters = cms.EDProducer("HITrackClusterRemover",
+     clusterLessSolution = cms.bool(True),
```

Comment:
about 80% of this PSet is identical to the hiMixedTripletStepClusters.
Please define a default and reuse it.